auth/src/test/java/io/grpc/auth/GoogleAuthLibraryCallCredentialsTest.java

```diff
@@ -401,6 +403,11 @@ public void jwtAccessCredentialsInRequestMetadata() throws Exception {
         || "https://example.com:123/a.service".equals(payload.get("aud")));
     assertEquals("test-email@example.com", payload.get("iss"));
     assertEquals("test-email@example.com", payload.get("sub"));
+
+    assertTrue(callCredentials.creds instanceof ServiceAccountJwtAccessCredentials);
```

Comment:
The rest of this test is verifying that auth is working like we expect. Would it be better to do the same here (instead of instanceof, etc)?
```java
Metadata.Key<String> quotaProject = Metadata.Key
    .of("X-Goog-User-Project", Metadata.ASCII_STRING_MARSHALLER);
assertEquals("test-quota-project-id", Iterables.getOnlyElement(headers.getAll(quotaProject)));
```